VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/tests/individual-participant-status-list.component.availability.spec.ts

```diff
@@ -48,20 +50,28 @@ describe('IndividualParticipantStatusListComponent Participant Status and Availa
 
         videoWebService = jasmine.createSpyObj<VideoWebService>('VideoWebService', ['getObfuscatedName', 'getCurrentParticipant']);
         videoWebService.getObfuscatedName.and.returnValue('t***** u*****');
-        videoWebService.getCurrentParticipant.and.returnValue(
-            Promise.resolve(
-                new LoggedParticipantResponse({
-                    participant_id: conference.participants[2].id,
-                    display_name: 'Jonh Doe',
-                    role: Role.Judge
-                })
-            )
-        );
+        logged = new LoggedParticipantResponse({
+            participant_id: conference.participants[2].id,
+            display_name: 'Jonh Doe',
+            role: Role.Judge
+        });
+        videoWebService.getCurrentParticipant.and.returnValue(Promise.resolve(logged));
```

Comment:
Does the spec still need `getCurrentParticipant` to be set up anymore now that the component gets the data from via the resolve guard now?

Reply:
Removed